### ISSUE TITLE
enable blind review

### ISSUE THREAD

**Issue by lsmith77** [opencfp/opencfp#1075]
this is a feature request to make it possible to do blind reviews of talks, ie. where the reviewer of submissions doesn’t see who submitted the talk. note I don’t run any conference myself but I think it would be good to make it easy for conferences to be able to do this.

that being said I know some conferences that require speakers to do 2 talks in order to be able to afford paying for travel expenses, maybe this should also be taken into account, ie. making it still possible to see if a prospective speaker submitted two interesting talks while making the final decisions on the schedule. but this is obviously a second step in this feature and maybe its not even relevant enough to even bother.

**Comment by localheinz** [comment]
@lsmith77 

Great idea, like it a lot!

**Comment by chartjes** [comment]
@lsmith77 Thanks so much for the suggestion.

There is a plan to work on OpenCFP Central (OpenCFP 2.0 will allow publishing of talks and speaker info to it with a paid API token) starting in March with a much-improved reviewing UX and UI, so I will add that to the list of features to be considered for that.

**Comment by ustayready** [comment]
We at Bsides Orlando are trying to roll out OpenCFP this year, biggest request among us was blind review. Looking forward to this!

**Comment by BackEndTea** [comment]
For last years DPC we forked opencfp to add this kind of functionality. We added a custom review 'module' that allowed reviewers to cycle through talks, while having the option to hide certain information. For example, hiding all speaker details. 

Unfortunately due to the architecture of opencfp having changed quite a bit since then it will be hard to merge this feature back in its current state.
However, if you are interested, you are free to look at our implementation, and rebuild it (hopefully a bit cleaner than my version), for the current architecture of opencfp.

If you have any questions about its implementation, feel reach out to me.
